src/hotspot/share/classfile/vmClassMacros.hpp

```diff
@@ -91,6 +91,7 @@
   do_klass(ThreadGroup_klass,                           java_lang_ThreadGroup                                 ) \
   do_klass(BasicVirtualThread_klass,                    java_lang_BaseVirtualThread                           ) \
```

Comment:
It should be BaseVirtualThread_klass rather BasicVirtualThread_klass (probably my fault when adding the alt implementation).

Reply:
Fixed.